docs/playbook-reference/actions/scans.rst

```diff
@@ -139,16 +139,35 @@ Troubleshooting Popeye
 
 Popeye scans run as Jobs in your cluster. If there are issues with a scan, troubleshoot as follows:
 
+events
+"""""""""""
 * To find errors with the Popeye job run:
 
 .. code-block:: bash
     :name: cb-popeye-get-events
 
     kubectl get events --all-namespaces --field-selector=type!=Normal | grep popeye-job
 
+Logs
+"""""""""
 * Additional errors can sometimes be found in the Robusta runner logs:
 
 .. code-block:: bash
     :name: cb-popeye-get-logs
 
     robusta logs
+
+
+Known issues:
```

Comment:
Remove `:` and make this a header too, with each individual issue a subsection/subheader.